### PR TITLE
Pack HTTP library and remove repetitive JARs

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2492

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -12,7 +12,6 @@ protobufJavaVersion=3.9.1
 jknackHandlebarsVersion=4.0.6
 nettyVersion=4.1.71.Final
```
Comment (daneshk) [inline gradle.properties:13]:
```suggestion
```

File: gradle.properties [hunk 2]
```diff
@@ -12,7 +12,6 @@ protobufJavaVersion=3.9.1
...
 picocliVersion=4.0.1
-nettyTcnativeVersion=2.0.46.Final
 bouncycastleVersion=1.69
 wso2CommonsPoolVersion=1.5.6.wso2v1
```
Comment (daneshk) [inline gradle.properties:16]:
```suggestion
```

Comment (BuddhiWathsala) [reply gradle.properties]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/598/commits/d192cd4e00a73effa84122cc36cd949d75457216

**Review by daneshk — APPROVED** [review]
LGTM

**Comment by shafreenAnfar** [comment]
LGTM!

**Pull request merged**